build fail looks real (i wonder if it explicitly tests that these are removed)
```
======================================================================
FAIL: test_translation (corehq.apps.app_manager.tests.test_bulk_ui_translation.BulkUiTranslation)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/apps/app_manager/tests/test_bulk_ui_translation.py", line 66, in test_translation
    u'home.start.demo': u'Dong'}
AssertionError: {u'fra': {u'entity.sort.title': u'wubble', u'unknown_string': u'Dong', u'home.st [truncated]... != {u'fra': {u'entity.sort.title': u'wubble', u'home.start.demo': u'Dong', u'activi [truncated]...
  {u'en': {u'date.tomorrow': u'wobble',
           u'entity.sort.title': u'wabble',
-          u'home.start.demo': u'Ding',
+          u'home.start.demo': u'Ding'},
?                                     +
-          u'unknown_string': u'Ding'},
   u'fra': {u'activity.locationcapture.Longitude': u'wibble',
            u'entity.sort.title': u'wubble',
-           u'home.start.demo': u'Dong',
?                                      ^
+           u'home.start.demo': u'Dong'}}
?                                      ^^
-           u'unknown_string': u'Dong'}}
----------------------------------------------------------------------
```

--- a/corehq/apps/app_manager/tests/test_bulk_ui_translation.py
+++ b/corehq/apps/app_manager/tests/test_bulk_ui_translation.py
@@ -58,12 +58,15 @@ class BulkUiTranslation(SimpleTestCase):
                 u'en': {
                     u'date.tomorrow': u'wobble',
                     u'entity.sort.title': u'wabble',
-                    u'home.start.demo': u'Ding'
+                    u'home.start.demo': u'Ding',
+                    u'unknown_string': u'Ding',
                 },
                 u'fra': {
                     u'activity.locationcapture.Longitude': u'wibble',
                     u'entity.sort.title': u'wubble',
-                    u'home.start.demo': u'Dong'}
+                    u'home.start.demo': u'Dong',
+                    u'unknown_string': u'Dong',
+                }
             }
 
         )

